app/components/wallet/transactions/Transaction.js

```diff
@@ -83,6 +83,14 @@ const messages = defineMessages({
     id: 'wallet.transaction.address.to',
     defaultMessage: '!!!To address',
   },
+  addressType: {
+    id: 'wallet.transaction.address.type',
+    defaultMessage: '!!!Address Type',
+  },
+  amount: {
```

Comment:
Try and reuse existing messages instead of creating new ones. Otherwise, it ends up doubling the translation work. (see global-messages.js for a place where we usually dump these)